scripts/wildfly/install-wildfly.sh

```diff
@@ -21,7 +21,7 @@ export PATH=$PATH:$(pwd)/galleon/bin
 
 echo ">>> Installing WildFly ..."
 rm -fr $WILDFLY_SERVER_DIR
-galleon.sh install wildfly#$WILDFLY_VERSION --dir=$WILDFLY_SERVER_DIR --layers=jaxrs-server,microprofile-health,microprofile-fault-tolerance,elytron-oidc-client,metrics
+galleon.sh install wildfly#$WILDFLY_VERSION --dir=$WILDFLY_SERVER_DIR --layers=jaxrs-server,microprofile-health,microprofile-fault-tolerance,elytron-oidc-client,metrics,opentelemetry
```

Comment:
don't you want to pull these layers form the `pom.xml` as your comment mentions?

Reply:
That's not possible (if I understand you correctly) since they serve very different purposes:
- `install-wildfly.sh` is only used locally to install WildFly on your local machine to be able to run ZAC in WildFly from IntelliJ
- pom.xml is only used to build the ZAC Docker Image where ZAC is packaged into a bootable WildFly JAR into a Docker image

You either use one or the other. Never both.